Yoast/YoastSEO.js
# Add verb inflection for English

## Summary
This PR can be summarized in the following changelog entry:

* Implements morphological analysis for keyword verbs in English.

## Relevant technical choices:
* For every English verb build its possible verb forms using regexes and a list of exceptions.

## Test instructions
This PR can be tested by following these steps:

* Add verb forms of nouns to `/spec/morphology/englishVerbSpec.js` and check if `yarn test` passes
* More proper testing will be possible after #1501 is merged.

Fixes #1516 Fixes #1514
